assets/css/custom.css

```
@@ -30,15 +30,726 @@
+
+.hero-aura-secondary {
+  right: -12%;
+  background: radial-gradient(circle, color-mix(in srgb, var(--color-tertiary2) 14%, transparent) 0%, transparent 70%);
+}
+
+/* Uses the same `container` constraint as the content sections below, so the
+   cards' outer edges line up with the "Take the security…" section at every
+   width (the `gap-*` matches the other multi-column sections). */
+.hero-split-panels {
+  @apply container relative grid grid-cols-1 md:grid-cols-2 gap-8 md:gap-12;
+  align-items: stretch;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)